build.gradle

```diff
@@ -57,6 +57,7 @@ task removeUnsupported(type: Delete) {
   dependsOn extractRelease
   delete "${jarDir}/${pkgDir}/aix-ppc"
   delete "${jarDir}/${pkgDir}/aix-ppc64"
+  delete "${jarDir}/${pkgDir}/linux-arm"
```

Comment:
Agree. I actually had already added it back. If you look at the sum of the two commits it's just one deletion. I'll squash them together.